magento/magento2
# Use repository to load order when manually creating an invoice

### Description (*)
Use repository to load order when manually creating an invoice. This way extension attributes are set and can be used later in the flow (e.g. observers).

### Fixed Issues (if relevant)
1. None I could find.

### Manual testing scenarios (*)
1. Create extension attribute on Order entity.
2. Place order and manually capture the invoice.
3. Order + extension attributes should be loaded but they're not.

### Contribution checklist (*)
